src/types.ts

```diff
@@ -0,0 +1,49 @@
+import type { Point, LineString, Polygon } from "geojson";
+
+// This describes the full structure of the GeoJSON we manage. We constrain the
+// default GeoJSON types and specify feature properties.
+export interface Scheme {
+  type: "FeatureCollection";
+  features: FeatureUnion[];
+  // Foreign members
+  scheme_name?: string;
+  authority?: string;
+  origin?: string;
+}
+
+// TODO Can we use a wildcard type, like Feature<? extends Geometry>
```

Comment:
I could not figure how to say "an array of a Feature, where the specific Geometry type may vary for each" in TS yet, so this is the hack